universal-login-wallet/cypress/plugins/index.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
```

Comment:
Try to add `universal-login-wallet/cypress` directory to include section in local `tsconfig.json` that might remove the need to add these annotations